src/Microsoft.OpenApi/Models/OpenApiLicense.cs

```diff
@@ -29,6 +29,21 @@ public class OpenApiLicense : IOpenApiSerializable, IOpenApiExtensible
         /// </summary>
         public IDictionary<string, IOpenApiExtension> Extensions { get; set; } = new Dictionary<string, IOpenApiExtension>();
 
+        /// <summary>
+        /// Parameterless constructor
+        /// </summary>
+        public OpenApiLicense() {}
+
+        /// <summary>
+        /// Initializes a copy of an <see cref="OpenApiLicense"/> object
+        /// </summary>
+        public OpenApiLicense(OpenApiLicense license)
+        {
+            Name = license.Name;
+            Url = license.Url;
```

Comment:
```suggestion
            Url = new Uri(license.Url,"");
```
I think this will work.  If not use new Uri(license.Url.OriginalString)